code/_onclick/hud/action.dm

```diff
@@ -135,12 +135,12 @@
 	var/image/img
 	if(owner.action_type == AB_ITEM && owner.target)
 		var/obj/item/I = owner.target
-		img = image(I.icon, src , I.icon_state)
+		overlays += I.appearance
```

Comment:
No; appearance grabs a copy of an atom's entire icon, overlays and underlays included, whereas an image object is limited to just an icon_state